lib/rules/selector-max-combinators/__tests__/index.js

```diff
@@ -242,11 +282,17 @@ testRule({
 					message: messages.expected('foo + bar', 0),
 					line: 1,
 					column: 1,
+					endLine: 1,
+					endColumn: 10,
 				},
 				{
 					message: messages.expected('foo + bar .aa', 0),
 					line: 1,
 					column: 29,
+					endLine: 1,
+					endColumn: 35,
+					// TODO: The following case is actually correct. It's hard to get the original selector before resolved.
```

Comment:
Shouldn't you create a github issue for it, instead of several TODOs?

Reply:
It's a good idea to create an issue and I'll do it.

But I think it's also worth leaving TODOs because we tend to forget an issue in the many current issues.

Reply:
I've opened an issue #6234.